doc/source/cluster/vms/references/ray-cluster-configuration.rst

```diff
@@ -129,6 +129,8 @@ Provider
             :ref:`location <cluster-configuration-location>`: str
             :ref:`resource_group <cluster-configuration-resource-group>`: str
             :ref:`subscription_id <cluster-configuration-subscription-id>`: str
+            :ref:`msi_name <cluster-configuration-msi-name>`: str
+            :ref:`msi_resource_group <cluster-configuration-msi-resource-group>`: str
```

Comment:
@mjd3 I think you'll need to also add the corresponding sections in the doc and write descriptions of the fields, do you mind double checking this?

Reply:
E.g. https://docs.ray.io/en/latest/cluster/vms/references/ray-cluster-configuration.html#provider-subscription-id

Reply:
Oops thanks for the catch! I _think_ I updated them correctly this time; let me know if something is missing still.